api/interfaces/requests.go

```diff
@@ -10,3 +12,13 @@ type LogInForm struct {
 	Email    string `json:"email"`
 	Password string `json:"password"`
 }
+
+type ValidationCode struct {
+	Email          string    `json:"email"`
+	ValidationCode string    `json:"validationCode"`
+	TimeExpiration time.Time `json:"timeExpiration"`
+}
+
+type Email struct {
```

Comment:
(Repeated) Please, follow the xForm / xReq "standart", e.g. `RequestCodeForm`. Feel free to use a better name, but I think something like that can be more clearer.